tests/data/col-681-part1.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There is obviously the risk that this fails if KK changes the output. But I'd say it is better to have a false negative than no tests.